adapter/src/components/AuthBoundary.js

```diff
@@ -13,9 +13,11 @@ const settingsQuery = {
 
 export const AuthBoundary = ({ url, children }) => {
     const { loading, error, data } = useDataQuery(settingsQuery)
-    useLocale(data && data.userSettings.keyUiLocale)
+    const locale = useLocale(
+        data && (data.userSettings.keyUiLocale || window.navigator.language)
```

Comment:
Is this wise?
I would have expected the DHIS2 user-locale to take precedence over the browser language.

Reply:
Did you perhaps mean to write:
```js
(data && data.userSettings.keyUiLocale) || window.navigator.language
```